main.go

```diff
@@ -29,17 +30,46 @@ func NewJoke(id, title, body string, score int) Joke {
 	return Joke{id, title, body, score}
 }
 
+// Page struct
+
+type Page struct {
+	Skip    int
+	Seed    int
+	Content []Joke
+}
+
+//GLOBAL VARIABLES
+
 var jokes []Joke
 
+var t *template.Template
+
 // HANDLERS
 
 func getJokes(w http.ResponseWriter, r *http.Request) {
-	t, err := template.ParseFiles("templates/index.html", "templates/header.html", "templates/footer.html")
+	var page Page
```

Comment:
it's better to declare variable at the place where it is used, on line 64. It could look like: ```page := Page{Skip:....```